repose-aggregator/src/docs/asciidoc/services/index.adoc

```diff
@@ -15,11 +15,11 @@ Enables the local Repose node or an entire cluster to store various types of dat
 The HTTP Connection Pool service is used for all outbound communication, and allows you to specify a number of things about those connections.
 
 == <<metrics.adoc#, Metrics Service>>
-Enables the registration, collection, and reporting of metrics across Repose.
-Such metrics provide insight into the state of Repose and its components at any given time.
+Enables the registration, collection, and reporting of metrics across *Repose*.
+Such metrics provide insight into the state of *Repose* and its components at any given time.
 
 == <<phone-home.adoc#,Phone Home Service>>
-*IN PROGRESS*
+Sends usage data consisting of the filters and services in use along with contact information to the *Repose Team*.
```

Comment:
Since this is just a quick description of, and reference to, the service page, I probably wouldn't include the specifics of what data is sent. After all, that might change, and we probably don't want to have to update it here **and** on the service page.

Reply:
Makes sense.

Reply:
FIxed.